x-pack/plugins/actions/server/builtin_action_types/swimlane/schema.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+export const ConfigMap = {
+  id: schema.string(),
+  key: schema.string(),
+};
+
+export const ConfigMapSchema = schema.object(ConfigMap);
+
+export const ConfigMapping = {
+  alertSourceKeyName: ConfigMapSchema,
+  severityKeyName: ConfigMapSchema,
+  caseNameKeyName: schema.nullable(ConfigMapSchema),
+  caseIdKeyName: ConfigMapSchema,
+  alertNameKeyName: ConfigMapSchema,
+  commentsKeyName: schema.nullable(ConfigMapSchema),
+};
+
+export const ConfigMappingSchema = schema.object(ConfigMapping);
+
+export const SwimlaneServiceConfiguration = {
+  apiUrl: schema.string(),
+  appId: schema.string(),
+  mappings: ConfigMappingSchema,
+};
+
+export const SwimlaneServiceConfigurationSchema = schema.object(SwimlaneServiceConfiguration);
+// secrets definition
+
+export const SwimlaneSecretsConfiguration = {
+  apiToken: schema.string(),
+};
+
+export const SwimlaneSecretsConfigurationSchema = schema.object(SwimlaneSecretsConfiguration);
+
+export const ExecutorSubActionSchema = schema.oneOf([
+  schema.literal('application'),
+  schema.literal('createRecord'),
```

Comment:
For the connector to be able to work through cases you need have a subaction called [`pushToService`](https://github.com/swimlane/kibana/blob/field_mapping_connector/x-pack/plugins/actions/server/builtin_action_types/servicenow/schema.ts#L74) with two fields `incident` and `comments`. The `incident` can contain arbitrary Swimlane related fields and the `comments` as [here](https://github.com/swimlane/kibana/blob/field_mapping_connector/x-pack/plugins/actions/server/builtin_action_types/servicenow/schema.ts#L42). Is the connector intent it to be used though Cases?